travis_configure.sh

```diff
@@ -62,8 +62,9 @@ before_install()
 # Returns 0
 enable_mysqlbinlog()
 {
-    sudo sed -i -e 's/^# *server.id.*$/server_id=1/g' $1
-    sudo sed -i -e 's/^# *log.bin.*$/log_bin=mysql-bin/g' $1
+    echo '[mysqld]' | sudo bash -c "cat >> $1"
```

Comment:
unfortunately not, as mentioned in this [post](https://www.cyberciti.biz/faq/sudo-append-data-text-to-file-on-linux-unix-macos/)